src/views/Viewpoint/DetailViewpoint/SidebarTabsContent/DescriptionTab/index.js

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
-import { Button, Classes } from '@blueprintjs/core';
 
 import { useTranslation } from 'react-i18next';
 import CollapsiblePanel from '../../../../../components/CollapsiblePanel';
 
 import { connectViewpointProvider } from '../../../context';
-import DownloadButton from '../../../../../components/DownloadButton';
 
-const iconMapping = {
-  image: 'media',
-  xls: 'th',
+const labelFromDocument = ({ key, properties: { label } }) => label || key;
+
+const filenameFromDocument = ({ key, document, properties: { label } }) => {
+  // source: https://stackoverflow.com/questions/190852/how-can-i-get-file-extensions-with-javascript
+  const ext = document.slice((Math.max(0, document.lastIndexOf('.')) || Infinity) + 1);
+  return `${label || key}.${ext}`.replace(' ', '_').toLocaleLowerCase();
```

Comment:
if `ext` equals to an empty string, your filename document should ends with a dot. Is it an expected behavior?

Reply:
Fixed